src/utils/vacation-days-utils.tsx

```diff
@@ -16,21 +16,21 @@ export const renderVacationDaysTextForCard = (user: User) => {
       <Grid>
         <Grid container spacing={2} alignItems="center">
           <Grid item xs={6}>
-            {strings.vacationsCard.spentVacations}
+            {strings.vacationsCard.vacationDays}
           </Grid>
           <Grid item xs={6}>
             <Typography color={spentVacationsColor}>
-              {/*{user.spentVacations}*/}
+              {parseVacationDays(user.attributes?.vacationDaysByYear ?? ["2024:30"])[new Date().getFullYear()]}
             </Typography>
           </Grid>
         </Grid>
         <Grid container spacing={2} alignItems="center">
           <Grid item xs={6}>
-            {strings.vacationsCard.unspentVacations}
+            {strings.vacationsCard.unspentVacationDays}
           </Grid>
           <Grid item xs={6}>
             <Typography color={unspentVacationsColor}>
-              {/*{user.unspentVacations}*/}
+              {parseVacationDays(user.attributes?.unspentVacationDaysByYear ?? ["2024:30"])[new Date().getFullYear()]}
```

Comment:
Similar changes here as mentioned previously, don't hardcode 2024 and maybe have a named variable for the currentyear